src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+
+function Pagination({ pageContext, type }) {
+  let { currentPage, totalPages, minPage, maxPage, path } = pageContext
+
+  const apiPageChange = value => {
+    if (value !== currentPage) pageContext.formData(pageContext.formResponse, value)
+  }
+
+  const direction = value => {
+    let count = value.target.innerText === 'More' ? 10 : -10
+    pageContext.setPageContext({
+      currentPage: currentPage,
+      totalPages: totalPages,
+      minPage: minPage + count,
+      maxPage:
```

Comment:
Remove nested ternary.

Reply:
Replaced nested ternary with maxPageFn that takes the argument count and returns maxPage